entity-framework/core/what-is-new/ef-core-7.0/plan.md

```diff
@@ -0,0 +1,342 @@
+
+## Theme: .NET platforms and ecosystem
+
+Much of the work planned for EF7 involves improving the data access experience for .NET across different platforms and domains. This involves work in EF Core where needed, but also work in other areas to ensure a great experience across .NET technologies. We will focus on the following platforms/technologies for the EF7 release:
+
+- [.NET MAUI](/dotnet/maui/what-is-maui)
+- [ASP.NET Core](https://dotnet.microsoft.com/learn/aspnet/what-is-aspnet-core)
+- [Azure Synapse](https://azure.microsoft.com/services/synapse-analytics/#overview)
+- [Blazor Server](https://dotnet.microsoft.com/apps/aspnet/web-apps/blazor)
+- [Blazor WebAssembly](https://dotnet.microsoft.com/apps/aspnet/web-apps/blazor)
+- [Windows Forms](/dotnet/desktop/winforms/overview)
+- [Windows Presentation Foundation (WPF)](https://visualstudio.microsoft.com/vs/features/wpf/)
+
+This list is based on many factors, including customer data, strategic direction, and available resources. The general areas we will be working on for these platforms are outlined below.
+
+### Distributed transactions
+
+Tracked by [Issue #715 in dotnet/runtime: Implement distributed/promoted transactions in System.Transactions](https://github.com/dotnet/runtime/issues/715)
+
+Value proposition: .NET Framework applications using distributed transactions can be ported to .NET 7.
+
+The `System.Transactions` library in .NET Framework contains native code that makes use of the Windows [Distributed Transactions Coordinator (DTC)](/previous-versions/windows/desktop/ms684146(v=vs.85)) to support distributed transactions. This code was never ported to .NET Core. In the .NET 7 timeframe, we plan to investigate and begin the process of bringing this functionality to modern .NET. This will be initially for Windows only, and will support only database scenarios where the ADO.NET provider also supports distributed transactions. Other uses of distributed transactions, such as in WCF, will not be supported in .NET 7. Based on feedback and cost, we may implement support for other scenarios and/or non-Windows platforms in a future release.
+
+### EF Core tooling
+
+Tracked by [Issue #26798: Modernize EF Core tooling](https://github.com/dotnet/efcore/issues/26798)
+
+Value proposition: `dotnet ef` commands are easy to use and work with modern platforms and technologies.
+
+.NET platforms have evolved since we first introduced tooling for migrations, database scaffolding, etc. in EF Core 1.0. In EF7, we plan to update the tooling architecture to better support new platforms, like .NET MAUI, and to streamline the process in areas such as the use of multiple projects.  This includes providing better feedback when things go wrong, better integration with logging, performance, and new [sugar](https://en.wikipedia.org/wiki/Syntactic_sugar).
```

Comment:
```suggestion
.NET platforms have evolved since we first introduced tooling for migrations, database scaffolding, etc. in EF Core 1.0. In EF7, we plan to update the tooling architecture to better support new platforms, like .NET MAUI, and to streamline the process in areas such as the use of multiple projects. This includes providing better feedback when things go wrong, better integration with logging, performance, and new [sugar](https://en.wikipedia.org/wiki/Syntactic_sugar).
```